hazelcast/src/main/java/com/hazelcast/sql/impl/SqlServiceImpl.java

```diff
@@ -151,9 +151,6 @@ public void reset() {
         if (jetSqlCoreBackend != null) {
             jetSqlCoreBackend.reset();
         }
-        if (internalService != null) {
```

Comment:
Wondering if we can leave an empty reset in case it might be used in the future, but probably not.
Also, I see there's a jetSqlCoreBackend but I can't find it. Is it used at all? cc @viliam-durina

Reply:
AFAIK it is used, the implementation of the Jet backend is located in the Jet repo.

Reply:
Ok, looks good, it's not using [`reset`](https://github.com/hazelcast/hazelcast-jet/blob/47d27e62075b7185d93838068affcae72109f7ae/hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/JetSqlCoreBackendImpl.java#L101-L103) to do something similar so all is well.